.github/workflows/actions/publish-to-pypi/action.yml

```diff
@@ -8,6 +8,9 @@ inputs:
   pypi-token:
     description: The pypi token to use for authentication
     required: true
+  dry-run:
+    description: Set to true for a test run that doesn't publish or tag.
+    default: false

```

Comment:
Tu devrais pas mettre ça à `true` par défaut au cas où quelqu'un fasse juste jouer avec/tester sans trop savoir ce qu'il fait?

Reply:
Hmmm, j'ai fait ça par habitude, car je me rappelle pas d'une fois où j'ai du dire à un programme "je suis sérieux cette fois" 🤔 la norme c'est que le service act, à moins que tu lui mentionne que c'est un troll run pis que faut pas foutre la marde.

🤣

Reply:
ceci dit, if you want me to change it I really don't mind.